### ISSUE TITLE
User data gets overwritten / doesn't work

### ISSUE THREAD

**Issue by Fragaile** [kaliiiiiiiiii/Selenium-Driverless#73]
the flag ``--user-data-dir`` works maybe once but gets overwritten next time code is run and goes back to default chrome. Maybe this is still not implemented, not sure :)

**Comment by Fragaile** [comment]
Basically I'm looking for a way to stay logged every time I launch selenium-driverless.

**Comment by kaliiiiiiiiii** [comment]
You'll have to:
1. set the `--user-data-dir=C://some_path/user_data`
2. Copy the data to another directory (or use `driver.quit(clean_dirs=False)` in the dev-branch version//next release
3. start the new session with the same `--user-data-dir=C://path_to_userdata_backup/user_data`

Hope that resolves your issue. Feel free to reportuin case it doesn't

**Comment by kaliiiiiiiiii** [comment]
@srichards2103 
With [`selenium-driverless==1.6`](https://github.com/kaliiiiiiiiii/Selenium-Driverless/discussions/75), you can now do:
```python
driver.quit(clean_dirs=False)
```

**Comment by elmissouri16** [comment]
> 2\. clean_dirs=False

> @srichards2103 With [`selenium-driverless==1.6`](https://github.com/kaliiiiiiiiii/Selenium-Driverless/discussions/75), you can now do:
> 
> ```python
> driver.quit(clean_dirs=False)
> ```

and is this works also on exceptions ?

**Comment by kaliiiiiiiiii** [comment]
> > 2. clean_dirs=False
> 
> > @srichards2103 With [`selenium-driverless==1.6`](https://github.com/kaliiiiiiiiii/Selenium-Driverless/discussions/75), you can now do:
> > ```python
> > driver.quit(clean_dirs=False)
> > ```
> 
> and is this works also on exceptions ?
@elmissouri16 
No, that's why I've introduced https://kaliiiiiiiiii.github.io/Selenium-Driverless/api/ChromeOptions.html#selenium_driverless.types.options.Options.auto_clean_dirs

for the case where you're using a context manager
Technically, `driver.quit` isn't even required in this case
